bibliotheca-hertziana-max-planck-institute-for-art-history.csl

```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" delimiter-precedes-last="never" et-al-min="4" et-al-use-first="1" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Bibliotheca Hertziana - Max Planck Institute for Art History</title>
+    <title-short>BH</title-short>
+    <id>http://www.zotero.org/styles/bibliotheca-hertziana-max-planck-institute-for-art-history</id>
+    <link href="http://www.zotero.org/styles/bibliotheca-hertziana-max-planck-institute-for-art-history" rel="self"/>
+    <link href="http://www.zotero.org/styles/bmj" rel="template"/>
+    <link href="https://www.biblhertz.it/3170244/rjbh_editorial_guidelines_en.pdf" rel="documentation"/>
+    <author>
+      <name>Chris Tomlinson</name>
+    </author>
+    <category citation-format="author-date"/>
+    <updated>2023-11-20T12:32:44+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="open-quote">“</term>
+      <term name="close-quote">”</term>
+      <term name="open-inner-quote">‘</term>
+      <term name="close-inner-quote">’</term>
+      <term name="page-range-delimiter">&#8211;</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" initialize="false" initialize-with=".">
+        <name-part name="given" text-case="capitalize-first"/>
+        <name-part name="family" text-case="capitalize-first"/>
+      </name>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" suffix=", ">
+      <label form="short" plural="never" text-case="lowercase" prefix=" " suffix=" "/>
+      <name sort-separator=" "/>
+    </names>
+  </macro>
+  <macro name="publisher">
+    <group>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="webpage post post-weblog" match="any">
+        <group delimiter=". ">
+          <text variable="URL"/>
+          <group>
+            <text term="accessed" text-case="capitalize-first" suffix=" "/>
+            <date variable="accessed">
+              <date-part name="month" suffix=" "/>
+              <date-part name="day" suffix=", "/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </if>
+    </choose>
+    <text variable="DOI" text-case="lowercase" prefix=" DOI: https://doi.org/"/>
+  </macro>
+  <macro name="title">
+    <group>
+      <choose>
+        <if type="book" match="any">
+          <group>
+            <text variable="title" font-style="italic" suffix=", "/>
+          </group>
+          <group delimiter="; ">
+            <date variable="original-date" prefix=" (" suffix="), ">
```

Comment:
you'd get this twice for books since it's also in the date-reference macro

Reply:
Date reference is used once in th ecitation and once in th ebibliography